extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -192,228 +459,61 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 				);
 
-				const migrationServiceTitle = this._view.modelBuilder.text().withProps({
-					value: constants.SQL_MIGRATION_SERVICE_DETAILS_HEADER(migrationService.name),
-					CSSStyles: {
-						'font-weight': 'bold'
-					}
-				}).component();
-
-				const connectionStatusLabel = this._view.modelBuilder.text().withProps({
-					value: constants.SERVICE_CONNECTION_STATUS,
-					CSSStyles: {
-						'font-weight': 'bold',
-						'width': '150px'
-					}
-				}).component();
-
-				const refreshStatus = this._view.modelBuilder.button().withProps({
-					label: constants.REFRESH,
-					secondary: true,
-					width: '50px'
-				}).component();
-
-
-
-				const connectionLabelContainer = this._view.modelBuilder.flexContainer().withLayout({
-					flexFlow: 'row',
-					alignItems: 'center'
-				}).withItems(
-					[
-						connectionStatusLabel,
-						refreshStatus
-					],
-					{
-						CSSStyles: { 'margin-right': '5px' }
-					}
-				).component();
-
-				const connectionStatus = this._view.modelBuilder.infoBox().component();
-				const connectionStatusLoader = this._view.modelBuilder.loadingComponent().withItem(connectionStatus).withProps({
-					loading: false
-				}).component();
-				refreshStatus.onDidClick(async (e) => {
-					connectionStatusLoader.loading = true;
-
-					const migrationService = await getSqlMigrationService(
-						this.migrationStateModel._azureAccount,
-						this.migrationStateModel._targetSubscription,
-						this.migrationStateModel._sqlMigrationService.properties.resourceGroup,
-						this.migrationStateModel._sqlMigrationService.location,
-						this.migrationStateModel._sqlMigrationService.name);
-					this.migrationStateModel._sqlMigrationService = migrationService;
-					const migrationServiceMonitoringStatus = await getSqlMigrationServiceMonitoringData(
-						this.migrationStateModel._azureAccount,
-						this.migrationStateModel._targetSubscription,
-						this.migrationStateModel._sqlMigrationService.properties.resourceGroup,
-						this.migrationStateModel._sqlMigrationService.location,
-						this.migrationStateModel._sqlMigrationService!.name);
-					this.migrationStateModel._nodeNames = migrationServiceMonitoringStatus.nodes.map((node) => {
-						return node.nodeName;
-					});
-
-					const state = migrationService.properties.integrationRuntimeState;
-					if (state === 'Online') {
-						connectionStatus.updateProperties(<azdata.InfoBoxComponentProperties>{
-							text: constants.SERVICE_READY(this.migrationStateModel._sqlMigrationService!.name, this.migrationStateModel._nodeNames.join(', ')),
-							style: 'success'
-						});
-					} else {
-						connectionStatus.updateProperties(<azdata.InfoBoxComponentProperties>{
-							text: constants.SERVICE_NOT_READY(this.migrationStateModel._sqlMigrationService!.name),
-							style: 'error'
-						});
-					}
-
-					connectionStatusLoader.loading = false;
+				this.migrationStateModel._nodeNames = migrationServiceMonitoringStatus.nodes.map((node) => {
```

Comment:
FYI you can shorten this to `.map(node => node.nodeName);` - no need for all that extra stuff.